### ISSUE TITLE
Quality benchmark

### ISSUE THREAD

**Issue by pinkforest** [openaustralia/righttoknow#837]
There are some requests that hurt everyone.

It would be great if there is some type of pressure to raise quality of submissions whilst not inpinging people's productivity.

It makes it harder to use the site given one user can effectively cause blockage (see the internal system) by the authority.

These blockages whilst illegal (maybe who knows) provide plausible deniability because one random person (or bot) did it.

Possible improvement options

1) Crowd-source reviews on first time posters by trusted ambassadors

2) Provide feedback via crowd-sourcing on first time posters from trusted ambassadors

3) Feedback positive-only score e.g. people can give thumbs up (not thumbs down possible as it can end up "brigading")

**Comment by benrfairless** [comment]
Hey @pinkforest 

Would you be able to provide examples of low quality requests?

If you would prefer, feel free to email these through.

**Comment by pinkforest** [comment]
https://www.righttoknow.org.au/user/john_smith_22

It's also against the house rules - targeting one minority group - if not plain vilification - it doesn't just ask stuff

**Comment by benrfairless** [comment]
I've been thinking about a possible way to address this without creating too much overhead on our volunteers.

We could consider immediately hiding anything that is reported to Administrators until we have time to review it. This would allow the request to proceed as normal, however would also reduce any harm by having the request public.

**Comment by benrfairless** [comment]
Hi @katska can I get you to review and provide your thoughts here?

**Comment by katska** [comment]
@benrfairless I don't agree with hiding everything reported, that's a policy which could easily have unintended chilling consequences. 
However where a simple call can't be made we have weighed up I temporarily hiding is appropriate and think that reviewing each request reported, and deciding case by case even to temporarily hide, while burdensome administratively, is a pragmatic and reasonable approach at small scale.
@pinkforest I do appreciate your concern - do you have anything else to consider which might help in this deliberation?

**Comment by katska** [comment]
I will mention that regardless of any merits the suggestions  @pinkforest offered so far require design and development considerations to action. Thank you @benrfairless in the first instance for pragmatic and practical suggestions to implement without requiring code changes.

A nontrivial solution doesn't appear obvious at this point

**Comment by katska** [comment]
Also @pinkforest also importantly if you see requests breaking the house rules please do continue to report them. 
While this *is* the way you an others in the community can lend a hand.
And if we need more eyes to keep on top, do consider if you know others who might also help?
